src/test/java/uk/gov/hmcts/reform/civil/service/DeadlinesCalculatorTest.java

```diff
@@ -323,4 +324,12 @@ private BankHolidays loadFixture() throws IOException {
         String input = ResourceReader.readString("/bank-holidays.json");
         return new ObjectMapper().readValue(input, BankHolidays.class);
     }
+
+    @Test
+    public void plusWorkingDays() {
```

Comment:
dont need public declaration

Reply:
also the test name should help describe the test scenario and should have a few that test different scenarios